modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` field in the revision template.
+
+Here is an example of `target` and `containerConcurrency` being used in a revision template.
+
+[source,yam]
+----
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: myapp
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/target: 50
+    spec:
+      containerConcurrency: 2
+      containers:
+      - image: myimage
+----
+
+[NOTE]
+`containerConcurrency` should only be used if there is a clear need to limit how many requests reach the application at a given time. Using `containerConcurrency` is only advised if the application needs to have an enforced constraint of concurrency.
+
+== Configuring concurrent requests using the target annotation
+
+The default target for the number of concurrent requests is `100`, but you can override this value by adding or modifying the `autoscaling.knative.dev/target` annotation value in the revision template.
+
+Here is an example of how this annotation would be used in the revision template to set the target to `50`.
+
+[source,yaml]
+----
+autoscaling.knative.dev/target: 50
+----
+
+== Configuring concurrent requests using the containerConcurrency field
+
+`containerConcurrency` sets a hard limit on the number of concurrent requests handled.
+
+[source,yaml]
+----
+containerConcurrency: 0 | 1 | 2-N
+----
+
+0:: means the system should decide the number of concurrent requests.
```

Comment:
It would be good to add a comment explaining how the system decided the number of concurrent requests. That was the first question I asked after reading this sentence.

Reply:
I think it's decided by the ConfigMap which we had consciously decided not to cover in this iteration of docs, but @markusthoemmes can confirm better. We can open a Jira for this sprint to cover ConfigMap specifically if you guys think that's worth doing now, we just didn't scope it for TP1 docs.

Reply:
0 means we allow unlimited requests today.

Reply:
Cool. So it would be good to mention this rather than "the system should decide the number of requests", to make it clear.